docs/api/consumerauthorization/authorization-token-generic-http.md

```diff
@@ -0,0 +1,182 @@
+# authorizationToken IDD
+**generic_http & generic_https**
+
+## Overview
+
+This page describes the generic_http and generic_https service interface of authorizationToken, which enables the verification of service consumption permissions on the provider system side, and the application of session-based service consumption control between the consumer and provider systems. An example of this interaction is when a consumer system with proper permissions obtains an expiring token of the appropriate type, which is then attached to its service consumption attempt. The provider system verifies the received token and performs the service operation only if the token has not expired and is valid for the actual service operation. Additionally, provider systems can choose to have the tokens generated for their services encrypted. Tokens for Event notification are also handled by this service in an event publisher/subscriber scenario. To enable other systems to use, to consume it, this service needs to be offered through the ServiceRegistry.
+
+The interfaces are implemented using protocol, encoding as stated in the following tables:
+
+## Interface Description
+
+**generic_http**
+
+Profile type | type | Version
+--- | --- | ---
+Transfer protocol | HTTP | 1.1
+Data encryption | N/A | -
+Encoding | JSON | RFC 8259
+Compression | N/A | -
+
+**generic_https**
+
+Profile type | type | Version
+--- | --- | ---
+Transfer protocol | HTTPS | 1.1
+Data encryption | TLS | -
+Encoding | JSON | RFC 8259
+Compression | N/A | -
+
+Hereby the **Interface Design Description** (IDD) is provided to the [authorizationToken – Service Description](../../assets/sd/5_0_0/authorization-token_sd.pdf). For further details about how this service is meant to be used, please consult that document.
+
+### generate
+
+The service operation **request** requires an [identity related header or certificate](../authentication_policy.md/#http) and an [AuthorizationTokenGenerationRequest](../data-models/authorization-token-generation-request.md) JSON encoded body.
+
+
+```
+POST /consumerauthorization/authorization-token/generate HTTP/1.1
+Authorization: Bearer <authorization-info>
+
+{
```

Comment:
in all examples, there should be spaces between the name and the value for easier reading. For example:
```
"tokenVariant": "USAGE_LIMITED_TOKEN_AUTH",
```